src/test/java/Homework23Test.java

```diff
@@ -0,0 +1,28 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pageFactory.HomePageFromFactory;
+import pageFactory.LoginPageFromFactory;
+
+public class Homework23Test extends BaseTest{
+
+@Test
+  public void createNewPlaylist () {
+      String expectedCreatedPlaylistMessage = "Created playlist \"Kristina.\"";
+      LoginPageFromFactory loginPageFromFactory = new LoginPageFromFactory(driver);
+      HomePageFromFactory homePageFromFactory = new HomePageFromFactory(driver);
+
+      loginPageFromFactory.provideEmail("kristina.sarkisyan@testpro.io")
+              .providePassword("o8URUDnW")
+              .clickSubmit();
+      homePageFromFactory.clickPlaylists()
+              .clickNewPlaylist()
+              .enterNewPlaylistName();
+String actualMsg = homePageFromFactory.getCreatedPlaylistMsg();
+      Assert.assertEquals(actualMsg,expectedCreatedPlaylistMessage);
+  }
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._